ui/StatusQ/src/StatusQ/Components/StatusGroupBox.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core 0.1
+
+
+GroupBox {
+    id: root
+
+    topPadding: label.implicitHeight
+    leftPadding: 16
+
+    /*!
+       \qmlproperty string StatusGroupBox::icon
+       This property holds the icon name for the icon represented on top of the
+       component as a title icon.
+    */
+    property url icon
```

Comment:
`string` or `url`? I think `string` is good enough (and more flexible in this case)